wetterdienst/provider/dwd/radar/util.py

```diff
@@ -46,3 +47,16 @@ def get_date_from_filename(filename):
         )
     except IndexError:
         pass
+
+
+def verify_hdf5(buffer: BytesIO):
+    import h5netcdf
+
+    buffer.seek(0)
+    try:
+        nc = h5netcdf.File(buffer, mode="r")
```

Comment:
Can't we use h5py.File here as well?

Reply:
Unfortunately not, because h5py apparently does not support reading from a File-like buffer.